api.planx.uk/modules/gis/service/classifiedRoads.ts

```diff
@@ -128,6 +128,7 @@ export const classifiedRoadsSearch = async (
             text: `is on a Classified Road`,
             data: features.map((feature: any) => ({
               name: `${feature.properties["RoadName1"]} - ${feature.properties["RoadClassification"]}`,
+              entity: feature.properties["GmlID"], // match Planning Data "entity" identifier for convenience when reporting inaccurate constraints
```

Comment:
This is a small change to address an "undefined" entity bug noted in the PlanningConstraints `utils.test.ts` file - this ensures `_overrides` will always have investigate-able values!